configs/mmdet3d/mono-detection/mono-detection_dynamic.py

```diff
@@ -0,0 +1,17 @@
+_base_ = ['./mono-detection_static.py']
+
+onnx_config = dict(
+    dynamic_axes={
+        'imgs': {
+            0: 'batch',
+            2: 'height',
+            3: 'width'
+        },
+        'cls_score': {
```

Comment:
what is the shape of `cls_score` and `bbox_pred`? Are dim `1` and `2` dynamic?

Reply:
outputs['cls_score'].shape  : torch.Size([1, 3, 96, 320])
 outputs['bbox_pred'].shape : torch.Size([1, 8, 96, 320])

Reply:
Are dim 2 and dim 3 dynamic? you can change the input shape and see if they change accordingly. If so, pls. add the dynamic dim in the config just like `input`.

Reply:
I tested with different input shape, the shape of cls_score and bbox_pred keep the smae.
That is 
results['bbox_pred'].shape
torch.Size([1, 8, 96, 320])
results['cls_score'].shape
torch.Size([1, 3, 96, 320])